mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -38,7 +38,7 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 	METHOD method_26160 asBlock ()Lnet/minecraft/class_2248;
 	METHOD method_26161 hasBlockEntity ()Z
 	METHOD method_26162 getLootTableId ()Lnet/minecraft/class_2960;
-	METHOD method_26403 getDefaultMaterialColor ()Lnet/minecraft/class_3620;
+	METHOD method_26403 getDefaultMapColor ()Lnet/minecraft/class_3620;
```

Comment:
What is "default" about this? Does it not just return the blocks map color as defined by the settings?

Reply:
It pulls the default blockstate for the color